android/build.gradle

```diff
@@ -17,16 +17,14 @@ buildscript {
 }
 
 ext {
-    kotlinVersion = "1.9.0"
-    buildToolsVersion = "33.0.2"
+    kotlinVersion = "1.9.24"
+    buildToolsVersion = "34.0.0"
     compileSdkVersion = 34
     minSdkVersion    = 24
     targetSdkVersion = 34
     supportLibVersion = "28.0.0"
-
-    // NDK is now entirely compatible with Apple Silicon M1 and M2 Macs as of version 24.0.
-    // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
     ndkVersion = "26.1.10909125"
+    hermesEnabled = "true"
```

Comment:
is this the right place?

Reply:
Yeah. Why?

Reply:
The log I added to check if hermes is enabled, checks out.

Reply:
Ah yeah I see the ext.